course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb

```diff
@@ -375,16 +385,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`standard_name` is restricted to be a CF standard name (see the [CF standard name table](http://cfconventions.org/standard-names.html)). If there is not a suitable CF standard name, `cube.standard name` is set to `None` and the `long_name` is used instead. `long_name` is less restrictive and can be set to be any string. `var_name` is less commonly required and is used to help distinguish the phenomena.\n",
-    "\n",
+    "`standard_name` is restricted to be a CF standard name (see the [CF standard name table](http://cfconventions.org/standard-names.html)).  \n",
+    "If there is not a suitable CF standard name, `cube.standard name` is set to `None` and the `long_name` is used instead.  \n",
+    "`long_name` is less restrictive and can be set to be any string.  \n",
+    "`var_name` is less commonly required and is used to help distinguish the phenomena."
```

Comment:
Might be worth just updating this to say something like
`var_name` is used when loading in a netCDF file. It represents the name of the variable in the file.